validation/src/test/java/io/serverlessworkflow/validation/test/WorkflowValidationTest.java

```diff
@@ -380,61 +380,61 @@ void testActionDefForEach() {
   public void testValidateRetry() {
     WorkflowValidator workflowValidator = new WorkflowValidatorImpl();
     List<ValidationError> validationErrors =
-            workflowValidator
```

Comment:
This file was just formatted, isnt it?

Reply:
The `mvn` command formats it, I believe.